cluster/CHAPTER.rst

```diff
@@ -63,24 +63,22 @@ Still, the usual rules of deterministic state machines apply: starting with the
 
 So, the distributed state machine technique ensures that the same operations occur on each host.
 But the problem remains of ensuring that every server agrees on the inputs to the state machine.
-This is a problem of *consensus*, and we'll address it with derivative of the Paxos algorithm.
+This is a problem of *consensus*, and we'll address it with a derivative of the Paxos algorithm.
 
-Consensus by Paxos
-==================
+## Consensus by Paxos
 
 Paxos was described by Leslie Lamport in a fanciful paper, first submitted in 1990 and eventually published in 1998, entitled "The Part-Time Parliament".
 Lamport's paper has a great deal more detail than we will get into here, and is a fun read.
 The references at the end of the chapter describe some extensions of the algorithm that we have adapted in this implementation.
 
 The simplest form of Paxos provides a way for a set of servers to agree on one value, for all time.
-MultiPaxos builds on this foundation by agreeing on a numbered sequence of facts, one at a time.
-To implement a distributed state machine, we use MultiPaxos to agree on each state-machine input, and execute them in sequence.
+Multi-Paxos builds on this foundation by agreeing on a numbered sequence of facts, one at a time.
+To implement a distributed state machine, we use Multi-Paxos to agree on each state-machine input, and execute them in sequence.
 
-Simple Paxos
-------------
+### Simple Paxos
 
 So let's start with "Simple Paxos", also known as the Synod protocol, which provides a way to agree on a single value that can never change.
-The name Paxos comes from the mythical island in "The Part-Time Parliament", where lawmakers vote on legislation through a process Lamport dubbed a Synod.
+The name Paxos comes from the mythical island in "The Part-Time Parliament", where lawmakers vote on legislation through a process Lamport dubbed the Synod protocol.
```

Comment:
I stand very much corrected - thanks :)

The Parliamentary protocol in section 3 is multi-decree, so very similar to MultiPaxos -- which means the Synod protocol is the correct term for this, then "Simple Paxos" section.